src/views/Catalog.vue

```diff
@@ -5,10 +5,36 @@
         <ion-title>{{ $t("Catalog") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-   
-    <ion-content :fullscreen="true">
-    
-    </ion-content>
+    <ion-content :fullscreen="true">    
+      <ion-searchbar></ion-searchbar>
+      <ion-row>
+        <ion-col>
+          <ion-card>
+            
+              <ion-card-header>
+                <img src="https://images.unsplash.com/photo-1523275335684-37898b6baf30?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8cHJvZHVjdHxlbnwwfHwwfHw%3D&w=1000&q=80" />
```

Comment:
Use Image component instead of using img tag.

Reply:
Done,Used image component instead.